scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -344,21 +345,23 @@
   cc/ComputeCluster
   (launch-tasks [this pool-name matches process-task-post-launch-fn]
     (let [task-metadata-seq (mapcat :task-metadata-seq matches)]
-      (log-structured/info "Launching tasks"
-                           {:pool pool-name
-                            :compute-cluster name
-                            :number-matches (count matches)
-                            :number-tasks (count task-metadata-seq)})
-      (let [futures
-            (doall
-              (map (fn [task-metadata]
-                     (.submit
-                       controller-executor-service
-                       ^Callable (fn []
-                                   (launch-task! this task-metadata)
-                                   (process-task-post-launch-fn task-metadata))))
-                   task-metadata-seq))]
-        (run! deref futures))))
+      (tracing/with-span [s {:name "launch-tasks" :tags
```

Comment:
Looks fine, but I'm not sure I like the naming scheme of `launch-tasks`. That could mean several things, especially as we add more spans across Cook.

Maybe something like `k8s.launch-tasks`, but I'm open to suggestions.

Also, from https://github.com/opentracing/specification/blob/master/semantic_conventions.md#span-tags-table  should we consider using the standardized `component` field of `cook.k8s`? 

I don't think we care about matches, just tasks, so I wouldn't bother to log them.